src/commands/diff.ts

```diff
@@ -57,20 +58,22 @@ export default class Diff extends Command {
     the non-null assertion '!' in this command.
     See https://github.com/oclif/oclif/issues/301 for details
   */
-  async run(): Promise<VersionResponse | void> {
+  async run(): Promise<WithDiff | void> {
     const { args, flags } = this.parse(Diff);
     /* eslint-disable-next-line @typescript-eslint/no-non-null-assertion */
     const [documentation, hub, token] = [flags.doc!, flags.hub, flags.token!];
 
-    if (args.FILE2) {
-      cli.action.start("* Let's compare the two given definition files");
-    } else {
-      cli.action.start(
-        "* Let's compare the given definition file with the currently deployed one",
-      );
+    if (flags.format == 'text') {
+      if (args.FILE2) {
+        cli.action.start("* Let's compare the two given definition files");
```

Comment:
I wouldn't use "let's do something", to me it tell me that we're going to do something together, human to human.
The following seems more neutral to me.
```suggestion
        cli.action.start("* Comparing the two given definition files");
```

Reply:
good point, thanks :+1: